communities/hacs/community-impact-proposal.md

```diff
@@ -0,0 +1,133 @@
+
+One step towards this has already been made as I have begun to create a [code of conduct](https://github.com/hacsbcu/conduct). It exists in its own repo and discussion around it can happen there. In extension to this I aim to have all this listed on pages on our website. Perhaps built from the repo source.
+
+This is loosely inspired by @mattburman [The longevity of a HackSoc](https://medium.com/hacksheffield/the-longevity-of-a-hacksoc-hackathon-society-4938fae20361)
+
+### Maintaining members
+
+Every year in September we gain a new cohort of first year students who attend the student union society fair. These first year students are excited to try new things, meet new people and make friends. Moving to university can be scary experience and joining a society is one of the ways to _fit in_ and rapidly make new friends and socialise.
+
+Because of this we receive upwards of 100 new signups to the group. The fact that we are a free society also contributes to this as there is 0 barrier to entry. However after the initial hype and excitement of joining university has disappeared we see reduced interaction at in person workshops and meetups. Our first social meetup the year is the most well attended. This is also echoed in lectures as the first lecture is the most well attended and then attendance drops off.
+
+We would like to therefore try harder to maintain the excitement and activity within the group. Im not quite sure how exactly we can go about doing that at this time, but I would like to look into it.
+
+> I propose we plan out our events up to a month in advance in order to be able to better market and promote our events to carry forward momentum from the first week of university.
+
+### Delivering our values
+
+One of our goals should always be to provide meaningful and valuable events and workshops to our members. Our community puts our *members* first. Everything we do should be for the benefit of our current and future members.
+
+We should aim to continue providing workshops, talks and socials for our members. Helping to increase their employability as well as providing them with a fun and safe social environment to interact in.
+
+Our values and goals include
+
+- Helping our members to become more employable
+- Helping our members to build
+- Helping our members to learn
+- Helping our members to make new friends
+
+In order to maintain these goals and values I propose
+
+> We continue to organise workshops and talks for our members. Introduce our members to more hackathons and organise trips to them. Facilitate a community of building and sharing to organically build projects in. Organise socials so that our members can interact face to face.
+
+In addition to the build point I'd like to make it easier for members to contribute directly to the society so for that reason
+
+> I propose we make our website more open and people can contribute to it and improve it.
```

Comment:
This one is less of a value but a consequence of a value. Just to yakshave a little